src/launcher/AppEntry.tsx

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { HashRouter as Router, Route, Redirect } from 'react-router-dom';
 import App from './App';
+import { Catalog } from './Catalog';
+import { Tutorial } from './Tutorial';
+import { Tasks } from './Tasks';
 
 const AppEntry = () => {
   return (
-    <Router basename="/mosaic/cloud-tutorials">
+    <Router basename="">
```

Comment:
This is needed to be hosted under ci.cloud.redhat.com/mosaic/cloud-tutorials, no?

Reply:
i've reverted this and switched back to BrowserRouter